WPFSKillTree/Model/Items/EquipmentData.cs

```diff
@@ -11,19 +11,20 @@ namespace POESKillTree.Model.Items
 {
     public class EquipmentData
     {
-        private readonly Options _options;
 
         public IReadOnlyDictionary<ItemType, IReadOnlyList<Affix>> AffixesPerItemType { get; private set; }
 
         public IReadOnlyList<ItemBase> BaseList { get; private set; }
 
         public IReadOnlyDictionary<string, ItemBase> BaseDictionary { get; private set; }
 
+        public ItemImageService ItemImageService { get; }
```

Comment:
Hm, I think this is used quite commonly in C#. It's much less confusing in an IDE than in the diff because class and instance are colored differently.